docs/lab/workflow/create-test.mdx

```diff
@@ -0,0 +1,125 @@
+2. The lab id.
+3. The collection method (`testkit`, `walk_in_test`, `at_home_phlebotomy`).
+4. The sample type (`dried_blood_spot`, `serum`).
+
+Once a Lab test is created and approved, you can use it when making a order.
+
+<Warning>
+  When you create a Lab Test, it is not immediately available. We need to
+  validate and approve it on our side, before you can use it when making orders.
+</Warning>
+
+### Example
+
+To begin, make a request to our [labs endpoint](/api-reference/lab-testing/labs), to get all available labs.
+
+```json Get all available labs
+[
+  {
+    "id": 1,
+    "slug": "labcorp",
+    "name": "LabCorp",
+    "first_line_address": "123 Main St",
+    "city": "San Francisco",
+    "zipcode": "91789"
+  }
+]
+```
+
+Once you've picked a lab, you can fetch all available biomarkers. This is done via our [biomarkers endpoint](/api-reference/lab-testing/biomarkers).
+
+```json Get all biomarkers
+{
+  "markers": [
+    {
+      "id": 1,
+      "name": "17-OH Progesterone LCMS",
+      "slug": "17-oh-progesterone-lcms",
+      "description": "17-OH Progesterone LCMS",
+      "lab_id": 1,
+      "provider_id": "070085",
+      "type": null,
+      "unit": null,
+      "price": "N/A"
+    },
+    {
+      "id": 2,
+      "name": "ABO Grouping",
+      "slug": "abo-grouping",
+      "description": "ABO Grouping",
+      "lab_id": 1,
+      "provider_id": "006056",
+      "type": null,
+      "unit": null,
+      "price": "N/A"
+    }
+  ],
+  "total": 2,
+  "page": 1,
+  "size": 2
+}
+```
+
+With this information, you can create a lab test using [this endpoint](/api-reference/lab-testing/post-test).
+
+As an example, let's say you wanted a test from `Labcorp`:
+
+```python Create a new test
+from vital import Client
+
+client = Client(api_key, "sandbox")
+
+data = client.LabTests.create_test(
+  name="Example test",
+  description="Example test",
+  lab_id=1,
+  method="at_home_phlebomoty",
+  sample_type="serum",
+  markers=[1, 2],
+)
+```
+
+Once your lab test creation is successful you will receive the following response:
+
+```json
+[
+  {
+    "lab_test": {
+      "name": "Example test",
+      "description": "Example test",
+      "sample_type": "serum",
+      "method": "at_home_phlebomoty",
+      "price": 10,
+      "is_active": false,
+      "lab": {
+        "slug": "labcorp",
+        "name": "LabCorp",
+        "first_line_address": "123 Main St",
+        "city": "San Francisco",
+        "zipcode": "91789"
+      },
+      "markers": [
+        {
+          "name": "17-OH Progesterone LCMS",
+          "slug": "17-oh-progesterone-lcms",
+          "description": "17-OH Progesterone LCMS"
+        },
+        {
+          "name": "ABO Grouping",
+          "slug": "abo-grouping",
+          "description": "ABO Grouping"
+        }
+      ]
+    }
+  }
+]
+```
+
+Do notice that `is_active` is set to `false`. You can verify its status by calling the [lab tests endpoint](/api-reference/lab-testing/tests).
```

Comment:
Can we join this and the previous warning in some way? Think it would make it clearer what the `is_active` flag is referring to when first creating a lab test.

Reply:
I think that would make it look weird since the active flag hasnt been addressed yet